utils/backup/S3/fo-restore-s3

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (C) Siemens AG 2017
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# version 2 as published by the Free Software Foundation.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source ${DIR}/before
+
+set -x
+
+echo Starting restore at `date`
+
+#### stop fossology scheduler
+echo "Stopping fossology scheduler..."
+/etc/init.d/fossology stop
+
+#### stop apache2
+echo "Stopping apache2..."
+/etc/init.d/apache2 stop
+
+echo Start removing old data at `date`
+
+${DIR}/../../fo-cleanold --delete-repository --delete-database
```

Comment:
You might need to ensure that the database is running.
This might delete productive data, you should display a warning and give the option to exit.